pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp

```diff
@@ -252,8 +252,8 @@ void Dialogs::CreateMemoryCardDialog::CreateControls()
 		.	SetToolTip(_t("Use at your own risk.  Erratic memory card behavior is possible (though unlikely)."))
 		.	SetInt(64),
 
-		RadioPanelItem(_("Folder [experimental]"), _("Store memory card contents in the host filesystem instead of a file."))
-		.	SetToolTip(_t("Automatically manages memory card contents so that the console only sees files related to the currently running software. Allows you to drag-and-drop files in and out of the memory card with your standard file explorer. This is still experimental, so use at your own risk!"))
+		RadioPanelItem(_("Folder [best with PCSX2]"), _("Transforms the memory card into an usable folder with files inside."))
+		.	SetToolTip(_t("Gives every game 8MB for the saves (essentially a whole memcard). You can see the structure and the saves with your File Explorer. Can be used to back-up individual gamesaves instead of all saves on a memcard. Incompatible with mymc(plus) which is a save manager.You can always convert back between folder type and single memcard file."))
```

Comment:
Maybe you can split the line code in to several so it's not one big line.

Reply:
8MB to 8 MB, so it matches the other changes.

Reply:
Noticed that PSX memcard says 1Mbit, should that be changed to 1MiB or 1MB?

Reply:
It's fine as it is, maybe just add space after 1.

Reply:
On the other hand change it, seems size is 1 MB.

Reply:
128 KiB (PSX) is what the radiopanel says, isn't that also wrong?

Reply:
Did a bit of search and original values are correct so leave them as they are for psx.

Reply:
( !it.IsPresent ? L"" : (it.IsPSX? pxsFmt( L"%u MBit", it.SizeInMB ) : ( it.SizeInMB > 0 ? pxsFmt( L"%u MiB", it.SizeInMB )
Yeah probably this

Reply:
Is this a different style? I don't know why it's not working.